### ISSUE TITLE
Not able to install PE 13 recovery

### ISSUE THREAD

**Issue by tommasoberlose** [PixelExperience/android-issues#5859]
### Build Date

PixelExperience_Plus_kebab-13.0-20231123-1705-OFFICIAL

### Device

kebab

### Version

thirteen_plus

### Describe the Bug

Not able to install the Pixel Experience 13 Plus Recovery for the OnePlus 8T starting from the Pixel Experience 12+.
Following the official installation guide when flashing the PE recovery and rebooting on recovery, I ended up in the Qualcomm crash dump mode. I was then able to reinstall the PE 12+ recovery to install the PE 12+ ROM.

The same error happened a few months ago with a previous build.

### Steps to Reproduce

Connect the device to the computer, and flash the additional images for the OnePlus 8T `vbmeta.img` and `dtbo.img`.
Then flash the recovery `PixelExperience_Plus_kebab-13.0-20231123-1705-OFFICIAL.img`, restarting the device in recovery mode will show the Qualcomm crash dump mode.

### Expected Behavior

Being able to install the PE 13+ recovery in order to install the Pixel Experience 13+.

**Comment by tommasoberlose** [comment]
@Broly1 Obviously I've followed all the steps in the wiki, I didn't want to report them in the issue description because it felt futile.
But opening the wiki and reading the main headlines I can confirm that
- I already have the bootloader unlocked, considering I have the PE 12+ installed; but I checked that.
- I flashed the compatible firmware, as reported.
- I flashed the recovery going through all the steps about restarting the device.

And I ended up with the Qualcomm crash dump mode.

It seemed to be related to some kind of partition problem 'cause using the TWRP recovery to check the system it was unable to mount partitions like `system_root`, etc... but I'm no expert here.
